install/kubernetes/cilium/templates/spire/agent/daemonset.yaml

```diff
@@ -27,10 +27,14 @@ spec:
       hostNetwork: true
       dnsPolicy: ClusterFirstWithHostNet
       serviceAccountName: {{ .Values.auth.mTLS.spire.install.agent.serviceAccount.name }}
-      {{- if gt (len .Values.auth.mTLS.spire.install.agent.initContainers) 0 }}
       initContainers:
-        {{- toYaml .Values.auth.mTLS.spire.install.agent.initContainers | nindent 8 }}
-      {{- end }}
+        - name: init
+          image: docker.io/library/busybox:1.35.0@sha256:223ae047b1065bd069aac01ae3ac8088b3ca4a527827e283b85112f29385fb1b
```

Comment:
should we have Renovate bot setup to update this image?

Reply:
Added a rule :)